README.md

```diff
@@ -919,7 +919,7 @@ steps:
 
 ### Post-rendering validation test (golden test)
 
-We use post-rendering validaton test to record and verify template rendering
+We leverage post-rendering validation tests to record(capture the anticipated outcome akin to expected output in unit test) and subsequently verify template rendering
```

Comment:
I think the original `use` is better than the updated `leverage`. `Leverage` is unnecessarily complex and abstract.

Reply:
done